### PR TITLE
Stable Video Diffusion

### PR DESCRIPTION
**Pull request by eaidova**
CVS-138880

### CODE REVIEW

File: notebooks/stable-video-diffusion/stable-video-diffusion.ipynb [hunk 1]
```diff
@@ -0,0 +1,1300 @@
+{
```
Comment (aleksandr-mokrov) [inline notebooks/stable-video-diffusion/stable-video-diffusion.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #435.                        noise_pred = noise_pred_uncond + guidance_scale * (noise_pred_cond - noise_pred_uncondsvd
</pre><p>Invalid syntax</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (aleksandr-mokrov) [inline notebooks/stable-video-diffusion/stable-video-diffusion.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #548.        if window_size % 2 == 0:svd
</pre><p>Invalid syntax</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (aleksandr-mokrov) [inline notebooks/stable-video-diffusion/stable-video-diffusion.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #60.            min_guidance_scale=min_guidnce_scale,
</pre><p>min_guidance_scale</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (aleksandr-mokrov) [inline notebooks/stable-video-diffusion/stable-video-diffusion.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #135.    demo.launch()
</pre><p> </p>

<pre class="ql-syntax" spellcheck="false">try:
    demo.queue().launch(debug=True)
except Exception:
    demo.queue().launch(debug=True, share=True)
# if you are launching remotely, specify server_name and server_port
# demo.launch(server_name='your server name', server_port='server port in int')
# Read more in the docs: https://gradio.app/docs/
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (eaidova) [reply notebooks/stable-video-diffusion/stable-video-diffusion.ipynb:1]: fixed

Comment (eaidova) [reply notebooks/stable-video-diffusion/stable-video-diffusion.ipynb:1]: fixed

Comment (eaidova) [reply notebooks/stable-video-diffusion/stable-video-diffusion.ipynb:1]: updated

Comment (eaidova) [reply notebooks/stable-video-diffusion/stable-video-diffusion.ipynb:1]: fixed

File: notebooks/stable-video-diffusion/stable-video-diffusion.ipynb [hunk 2]
```diff
@@ -0,0 +1,1296 @@
...
```
Comment (aleksandr-mokrov) [inline notebooks/stable-video-diffusion/stable-video-diffusion.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #436.    )
</pre><p>Remove this parenthesis</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

**Review by aleksandr-mokrov — APPROVED** [review]

**Pull request merged**